apps/opik-frontend/src/components/pages/ExperimentsPage/ExperimentRowActionsCell.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu";
+import { Button } from "@/components/ui/button";
+import { MoreHorizontal, Trash } from "lucide-react";
+import React, { useCallback, useRef, useState } from "react";
+import { CellContext } from "@tanstack/react-table";
+import ConfirmDialog from "@/components/shared/ConfirmDialog/ConfirmDialog";
+import { Experiment } from "@/types/datasets";
+import useExperimentBatchDeleteMutation from "@/api/datasets/useExperimentBatchDeleteMutation";
+
+export const ExperimentRowActionsCell: React.FunctionComponent<
+  CellContext<Experiment, unknown>
+> = ({ row }) => {
+  const resetKeyRef = useRef(0);
+  const experiment = row.original;
+  const [open, setOpen] = useState<boolean>(false);
+
+  const experimentBatchDeleteMutation = useExperimentBatchDeleteMutation();
+
+  const deleteExperimentsHandler = useCallback(() => {
+    experimentBatchDeleteMutation.mutate({
+      ids: [experiment.id],
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [experiment]);
```

Comment:
Maybe only `experiment.id`? I get it shouldn't mutate the `experiment` object anyway

Also about the `// eslint-disable-next-line react-hooks/exhaustive-deps` maybe we can just destruct the `useMutation` and use the `mutate` directly and that should not change the function reference:
```typescript
  const { mutate: deleteExperiments } = useExperimentBatchDeleteMutation();
  const { id: experimentId } = experiment;

  const deleteExperimentsHandler = useCallback(() => {
    deleteExperiments({ ids: [experimentId] });
  }, [deleteExperiments, experimentId]);
```

Reply:
@ferc Nice suggestions. I will change it with my next PR.